code_generator/bin/flutter_storyblok_code_generator.dart

```diff
@@ -43,6 +45,13 @@ class GenerateCommand extends Command {
       help: "Your Personal Access Token, not your Space access token",
       mandatory: true,
     );
+    argParser.addOption(
+      _optionLocation,
+      abbr: "l",
```

Comment:
`r` was already in use in rate limit so I chose `l` for location instead of region.